src/ProjectTemplates/Web.ProjectTemplates/RazorComponentsWeb-CSharp.csproj.in

```diff
@@ -9,7 +9,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Components.Prerendering" Version="${MicrosoftAspNetCoreMvcComponentsPrerenderingPackageVersion}" />
```

Comment:
There's also an outstanding PR that removes the other `<PackageReference>` to `NewtonsoftJson` (#9025).

I don't know which of the two PRs will get merged first, but if this one is the last to go in, could you update it to remove the surrounding `<ItemGroup>` wrapper, since after your change it will be entirely empty?

Reply:
I have to another PR that removes the `AddRazorSupportForMvc` property from templates. Let me address it as part of that PR (just so we don't spend a few hours for a small-ish fix)

Reply:
OK thanks! Also, #9025 was now merged already.

Reply:
Actually, nevermind. Have to resolve merge conflicts. I'll clean it up as part of that 👍